docs/changelog/112319.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 112319
+summary: Capabilities to Hide/Show Vector fields from Source
+area: "Mapping, Vector Search, Relevance"
```

Comment:
Will update to `Mapping` once the Pr is finalize

Reply:
Might as well do that now IMO